### PR TITLE
[Docs] add wasm-nginx-module

### PR DESCRIPTION
**Pull request by spacewander**
For https://github.com/WasmEdge/WasmEdge/pull/1037#issuecomment-1154654206
Signed-off-by: spacewander <spacewanderlzx@gmail.com>

### CODE REVIEW

File: docs/book/en/src/frameworks/mesh/wasm-nginx-module.md [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+
+### Step 2: Build the corresponding Wasm file
+
+```shell
+tinygo build -o ./fault-injection/main.go.wasm -scheduler=none -target=wasi ./fault-injection/main.go
+```
+
+### Step 3: Load and execute the Wasm file
+
+Build your Nginx with wasm-nginx-module included and WasmEdge shared library installed in the right path.
```
Comment (juntao) [inline docs/book/en/src/frameworks/mesh/wasm-nginx-module.md:27]: Can you provide a link or a paragraph on how to do this exactly? Do APISIX binary distributions have this already? Thanks!

Comment (spacewander) [reply docs/book/en/src/frameworks/mesh/wasm-nginx-module.md:27]: There is a section about setting up WasmEdge when building Nginx: https://github.com/api7/wasm-nginx-module#install-dependencies.

However, it is more for developers, which requires the reader to have Nginx C module development experience.

Comment (juntao) [reply docs/book/en/src/frameworks/mesh/wasm-nginx-module.md]: I made some changes. Please let me know if you agree. Thanks.

Comment (spacewander) [reply docs/book/en/src/frameworks/mesh/wasm-nginx-module.md]: LGTM

**Review by hydai — APPROVED** [review]

**Pull request merged**